src/main/java/edu/kit/hci/soli/service/impl/RoomServiceImpl.java

```diff
@@ -0,0 +1,75 @@
+package edu.kit.hci.soli.service.impl;
+
+import edu.kit.hci.soli.domain.Room;
+import edu.kit.hci.soli.repository.RoomRepository;
+import edu.kit.hci.soli.service.RoomService;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+import java.util.NoSuchElementException;
+
+/**
+ * Service class for managing {@link Room} entities.
+ * Provides methods to change and retrieve room details.
+ */
```

Comment:
If the interface is documented, the documentation of this class (mostly) becomes redundant